i18n.sh

```diff
@@ -3,8 +3,9 @@
 set -ev
 find config "views/" -name "*.ejs"| xargs xgettext --language=PHP -k_n:1,2 --force-po -o tt.pot --from-code=UTF-8 -
 find config -name "*.yml"| xargs xgettext --language=Python -k_n:1,2 --force-po -o tt.pot --from-code=UTF-8 -j -
-find bin "public/build/javascript/" -name "*.js"| xargs xgettext -k_n:1,2 --force-po -o tt.pot --from-code=UTF-8 -j -
+find bin src -name "*.js" | xargs xgettext -k_n:1,2 --force-po -o tt.pot --from-code=UTF-8 -j -
+find src -name "*.jsx" | xargs xgettext --language=Python -k_n:1,2 --force-po -o tt.pot --from-code=UTF-8 -j -
```

Comment:
Hackish: `xgettext` can't parse .jsx syntax, but it seems that the Python parser is permissive enough to extract translation keys from .jsx files.